config/riff-build.yaml

```diff
@@ -8,205 +8,201 @@ metadata:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
```

Comment:
this should still be present. You may need to update your local copy of controller-gen. If you delete you copy and run make, it will install the correct version.